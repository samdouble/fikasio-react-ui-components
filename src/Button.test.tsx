import React from 'react';
import renderer from 'react-test-renderer';
import { Button } from '.';

describe('Button', () => {
  it('Renders correctly', () => {
    const componentJSON = renderer.create(<Button.Action />).toJSON();
    expect(componentJSON).toMatchSnapshot();
  });
});
