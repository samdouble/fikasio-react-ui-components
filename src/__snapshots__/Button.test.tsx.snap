// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Button Renders correctly 1`] = `
<button
  className="fikasio-actionbutton fikasio-theme-dark btn btn-primary"
  disabled={false}
  onClick={[Function]}
  style={Object {}}
/>
`;
